app/pages/lab/project.cjsx

```diff
@@ -98,13 +98,13 @@ EditProjectPage = createReactClass
             <div className="nav-list-header">Need some help?</div>
             <ul className="nav-list">
               <li>
-                <Link className="nav-list-item" to="/help">Read a tutorial</Link>
+                <a className="nav-list-item" href="https://zooniverse.github.io/help">Read a tutorial</a>
```

Comment:
Links from project builder forms should probably use `target=_blank ref="noopener nofollow"` to avoid losing work in progress in the forms while referring to help.